infra/common/src/test/java/org/apache/shardingsphere/infra/metadata/database/ShardingSphereDatabaseTest.java

```diff
@@ -31,51 +43,117 @@
 import org.apache.shardingsphere.test.fixture.jdbc.MockedDataSource;
 import org.apache.shardingsphere.test.mock.AutoMockExtension;
 import org.apache.shardingsphere.test.mock.StaticMockSettings;
+import org.apache.shardingsphere.infra.config.database.DatabaseConfiguration;
+import org.apache.shardingsphere.infra.config.props.ConfigurationProperties;
+import org.apache.shardingsphere.infra.database.core.type.DatabaseType;
+import org.apache.shardingsphere.infra.database.core.type.DatabaseTypeRegistry;
+import org.apache.shardingsphere.infra.metadata.database.resource.StorageResource;
+import org.apache.shardingsphere.infra.metadata.database.schema.model.ShardingSphereSchema;
+
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.junit.jupiter.MockitoSettings;
 import org.mockito.quality.Strictness;
 
+import javax.sql.DataSource;
 import java.sql.SQLException;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.LinkedList;
 import java.util.Properties;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
-import static org.mockito.Mockito.mock;
+import java.util.HashMap;
+import java.util.Map;
 
 @ExtendWith(AutoMockExtension.class)
 @StaticMockSettings(ShardingSphereDatabase.class)
 @MockitoSettings(strictness = Strictness.LENIENT)
 class ShardingSphereDatabaseTest {
-    
+
+    @Test
+    void assertContainsSchema() {
+        DatabaseType databaseType = mock(DatabaseType.class);
+        RuleMetaData ruleMetaData = mock(RuleMetaData.class);
+        Map<String, ShardingSphereSchema> schemas = new HashMap<>();
+        ShardingSphereSchema schema = mock(ShardingSphereSchema.class);
+        schemas.put("schema1", schema);
+        ShardingSphereDatabase database = new ShardingSphereDatabase("foo_db", databaseType, mock(ResourceMetaData.class), ruleMetaData, schemas);
+        assertTrue(database.containsSchema("schema1"));
+        assertFalse(database.containsSchema("non_existent_schema"));
+    }
+
+    @Test
+    void assertGetSchema() {
+        DatabaseType databaseType = mock(DatabaseType.class);
+        RuleMetaData ruleMetaData = mock(RuleMetaData.class);
+        Map<String, ShardingSphereSchema> schemas = new HashMap<>();
+        ShardingSphereSchema schema = mock(ShardingSphereSchema.class);
+        schemas.put("schema1", schema);
+        ShardingSphereDatabase database = new ShardingSphereDatabase("foo_db", databaseType, mock(ResourceMetaData.class), ruleMetaData, schemas);
+        assertEquals(schema, database.getSchema("schema1"));
```

Comment:
Please use assertThat to assert

Reply:
ack

Reply:
Hi @zhaojinchao95 ,

I have a quick question regarding the spotless step in this repo, when I was executing `mvn spotless:apply` or `./mvnw spotless:apply`, I got error message saying that `spotless` plugin could not be found.

I am wondering am I missing anything to re-format the codes ? I am using VS Code + CodeSpaces currently